content/Arduino Cloud/Cloud Editor/If-your-board-is-not-detected-by-Arduino-Cloud-Editor.md

```diff
@@ -32,9 +32,7 @@ Learn how to troubleshoot possible issues when connecting a board to the Cloud E
 
 ### Classic Nano boards and boards with generic USB chips
 
-Classic Nano boards and some clone boards with generic USB chips will show up as **"Board"** in the board selector. When selecting a "Board" port you will be prompted to select the board type manually.
-
-![Cloud Editor with port name showing in the board selector ](img/WebEditor_port_no_board_name.png)
+Classic Nano boards and some clone boards with generic USB chips will show up as **"Board"** in the board selector. When selecting a **"Board"** port you will be prompted to select the board type manually.
```

Comment:
We should test this with the new Cloud Editor.

Reply:
![image](https://github.com/arduino/help-center-content/assets/83598143/97e1420c-975d-4ae1-ab13-33789b7ee29d)